projects/stargazer/plugins/other/radius/server.cpp

```diff
@@ -84,5 +100,41 @@ void Server::handleReceive(const error_code& error, const std::optional<RadProto
         printfd(__FILE__, "Error asyncReceive: the request packet is missing\n");
         return;
     }
+
     m_radius.asyncSend(makeResponse(*packet), source, [this](const auto& ec){ handleSend(ec); });
 }
+
+bool Server::findUser(const RadProto::Packet& packet)
+{
+    STG::User* user;
```

Comment:
`Server` is in the `STG` namespace, you don't need to prefix `User` with `STG::`.

Reply:
Also, please move it close to the point of use (`FindByName` call, I believe) and initialize with `nullptr`.